capsules/src/ft6x06.rs

```diff
@@ -147,12 +148,13 @@ impl<'a> i2c::I2CClient for Ft6x06<'a> {
                         + (buffer[touch_event * 8 + 5] as u16);
                     let pressure = Some(buffer[touch_event * 8 + 6] as u16);
```

Comment:
Ugh these embedded constants! I don't want to block a 2.0 update on fixing cruft like this, but fixing it seems pretty important from a maintenance perspective.

Reply:
I agree, but I'm not sure how to fix this. I can either:
- declare some consts instead of the hard coded numbers.
- have a struct instead of a buffer, but I'm not sure I can convert the struct to the array with some kind of `memcpy` that would be considered safe

Any suggestion is welcome.